### PR TITLE
Sign in / Sign up

### PR DESCRIPTION
**Pull request by pidip47**

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -1,4 +1,26 @@
 html,body {
   margin:0;
   padding:0;
+  font-family: "Roboto","Helvetica","Arial",sans-serif;
```
Comment (ManoloTonto1) [inline client/src/index.css]: Remove css pls

File: client/src/routes/login/Login.tsx [hunk 2]
```diff
@@ -1,30 +1,205 @@
...
+			{value === index && (
+				<Box sx={{ p: 3 }}>
+					{children}
+				</Box>
+			)}
+		</Box>
+	);
+}
+
+function a11yProps(index: number) {
```
Comment (ManoloTonto1) [inline client/src/routes/login/Login.tsx:41]: a11i props? maybe make it more self explanitory

File: client/src/routes/login/Login.tsx [hunk 3]
```diff
@@ -1,30 +1,205 @@
...
+	const signIn = (e: React.FormEvent<HTMLFormElement>) => {
+		e.preventDefault();
+		
+		// logs voor testen
+		console.log(email);
+		console.log(password);
+		console.log(rememberMe);
+	}
+	
+	const signUp = (e: React.FormEvent<HTMLFormElement>) => {
```
Comment (ManoloTonto1) [inline client/src/routes/login/Login.tsx:59]: gebruik een useCallback(()=>,[]) voor de functie. memoizes the method so it doen't have to redo the method.

File: client/src/routes/login/Login.tsx [hunk 4]
```diff
@@ -1,30 +1,205 @@
...
+		e.preventDefault();
+
+		let error: boolean = true;
+		if(password == confirmPassword) {
+			setPasswordsMatch(true);
+			error = false;
```
Comment (ManoloTonto1) [inline client/src/routes/login/Login.tsx:65]: add a return; and remove the else. makes it cleaner.

File: client/src/routes/login/Login.tsx [hunk 5]
```diff
@@ -1,30 +1,205 @@
...
+		} else {
+			setPasswordsMatch(false);
+			error = true;
+		}
+
+		if(!error) {
+
```
Comment (ManoloTonto1) [inline client/src/routes/login/Login.tsx:72]: you have eslint working? seems like the linter is no hatching this error.

File: client/src/routes/login/Login.tsx [hunk 6]
```diff
@@ -1,30 +1,205 @@
...
+	const [voornaam, setVoornaam] = React.useState("");
+	const [tussenvoegsel, setTussenvoegsel] = React.useState("");
+	const [achternaam, setAchternaam] = React.useState("");
+	
+	// errors
+	const [passwordsMatch, setPasswordsMatch] = React.useState(true);
+	
+
+	// data handlers
+	const handlePassword = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
```
Comment (ManoloTonto1) [inline client/src/routes/login/Login.tsx:110]: useCallback for all these.

**Review by ManoloTonto1 — CHANGES_REQUESTED** [review]
fix these pls

**Review by GeneDrive — CHANGES_REQUESTED** [review]
add gradient background like it was planned

**Review by GeneDrive — APPROVED** [review]
ziet er goed uit man, leuke achtergrond veel beter idee dan de gradient

File: client/src/context/UserContext.tsx [hunk 7]
```diff
@@ -37,13 +37,19 @@ export const UserProvider = ({ children }: $TSfixMeAny) => {
...
 		palette: {
-			mode: darkMode ? 'dark' : 'light',
+			mode: !darkMode ? 'dark' : 'light',
 			primary: {
 				main: yellow[500],
 			},
 			secondary: {
 				main: red['A700'],
 			},
+			// background: {
```
Comment (ManoloTonto1) [inline client/src/context/UserContext.tsx:47]: delete pls.

File: client/src/routes/login/Login.tsx [hunk 8]
```diff
@@ -1,29 +1,111 @@
...
+	return {
+		id: `simple-tab-${index}`,
+		'aria-controls': `simple-tabpanel-${index}`,
+	};
+}
 
 function Login() {
+	const [tab, setTab] = React.useState(0);
+	
+	// errors
```
Comment (ManoloTonto1) [inline client/src/routes/login/Login.tsx:58]: remove comments

File: client/src/routes/login/tabs/SignIn.tsx [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+	}, [password]);
+
+	const handleEmail = useCallback((e: React.ChangeEvent<HTMLTextAreaElement>) => {
+		setEmail(e.target.value);
+		console.log(e.target.value);
+	}, []);
+
+	const signIn = (e: React.FormEvent<HTMLFormElement>) => {
+		e.preventDefault();
+		// logs voor testen
```
Comment (ManoloTonto1) [inline client/src/routes/login/tabs/SignIn.tsx:35]: remove comments.

**Review by ManoloTonto1 — CHANGES_REQUESTED** [review]

**Review by ManoloTonto1 — APPROVED** [review]
MERGE THI BITCH

**Pull request merged**